src/components/SummaryDefineCompany/README.md

```diff
@@ -0,0 +1,6 @@
+# WARNING
+
+For some unknown reason, the SummaryDefineCompany component in this folder cannot exist in the ReviewConfirm/ folder where it logically belongs: it
+causes import errors in Summary.vue and lots of unit test errors.
+
+Therefore, as a workaround, the subject component exists in this folder.
```

Comment:
And now you know.

Only try to fix this if you have nothing better to do. I lost most of an afternoon on this :/